### ISSUE TITLE
WCAG2Mobile: Draft feedback - 1.1.1 Non-text content

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#640]
(Mary Jo) Regarding the note: "Not all mobile platforms provide a way to programmatically associate a label with non-text content." Need to add the following, which is consistent with other criteria that require something that may not be supported fully: "In such situations, it may not be possible for the non-web software to meet this success criterion." 

- **Recommendation - Need to add the following, which is consistent with other criteria that require something that may not be supported fully: "In such situations, it may not be possible for the non-web software to satisfy this success criterion.**"

(Chris L) 

1.  https://w3c.github.io/matf/#interpretation-of-web-terminology-in-a-mobile-context states set of screens, and calls out WCAG2ICT on set of software phrasing. Wouldn't the app itself be a set of software? I'm not following the logic they are raising here.

- [ ] Do we define a separate issue here on this aspect?

2.  Overall comment on use of screen/view terminology when this is under development in WCAG3 and not fully developed is concerning, i.e. The glossary terms “document” and “software” in WCAG2ICT are replaced with the defined terms “screen” and “view”. 

- **Recommendation -  Phil has filed https://github.com/w3c/wcag2ict/issues/638**

3. Closed functionality definition should be consistent across W3C, WCAG, WCAG2ICT and Mobile. Another related term should be defined if WCAG2ICT and Mobile2 vary in definition of term.

- **Recommendation -  WCAG2ICT should file an issue with WCAG2Mobile to define closed functionality.**

4.  If WCAG2Mobile doesn't differ from WCAG2ICT, it may be better that they just state that there is no difference rather than listing WCAG 2.x and WCAG2ICT in two different detail/collapsible content containers within the SC definition.

- **Recommendation -  WCAG2ICT should file an issue with WCAG2Mobile to state that there is no difference rather than listing WCAG 2.x and WCAG2ICT in two different detail/collapsible content containers within the SC definition.**

5. Add to the note "not all mobile platforms..." Add in When mobile platforms provide a way to programmatically associate a label with non-text content, that the platform provides that ability to the developer/designer of the app (product). 
- **Recommendation - WCAG2ICT should file an issue with WCAG2Mobile to align this phrasing**

**Comment by maryjom** [comment]
This issue seems to have several different topics that are beyond the SC 1.1.1 Non-text Content. Numbering my comments to match the numbering in the current issue description. For the first suggestion, that isn't numbered, I propose the WCAG2Mobile note reads as follows:

>Not all mobile platforms provide a way to programmatically associate a [label](https://www.w3.org/TR/WCAG22/#dfn-labels) with [non-text content](https://www.w3.org/TR/WCAG22/#dfn-non-text-content). In such situations, it may not be possible for the non-web software to satisfy this success criterion.

1. Perhaps we should open an issue stating that it is unclear whether a "set of software" even exists in the WCAG2Mobile interpretation. It seems that it doesn't - it's replaced by "set of views". Clarity is needed in WCAG2Mobile on this topic.
2. I think this part should be removed from this issue, but ensure the comment is covered fully in Issue 638. Then we can discuss and tracked use of "view" in issue 638 alone and keep this issue strictly for 1.1.1 Non-text Content comments.
3. I agree it is odd WCAG2Mobile says the closed functionality definition is different, but doesn't offer what the definition is.  I think it's best to spawn this point into an issue on its own. Additionally, they say the term "accessibility services of platform software" has been modified - and no information is given for that term either. The issue we open should say something like:
>In the Key Terms section paragraph 4, statements are made about WCAG2Mobile changing certain definitions, for example: 
>* "The term “accessibility services of platform software”, introduced by WCAG2ICT, has been modified to reflect its different use in mobile applications." Yet there is no content in the details of this section on the changes to the term. In WCAG2ICT (including the 2013 version), this definition is congruent with the definition of "platform accessibility services" in the U.S. Revised 508 Standards and the use of the term "platform accessibility services" in the EN 301 549, so its use and meaning is fairly well established. 
>* "Additionally, “closed functionality” has a different meaning in the context of mobile applications." However, the key term definition for "closed functionality" does not show what the different meaning is. 
>It is concerning that WCAG2Mobile intends to change WCAG2ICT definitions which are based on long-established terms found in the U.S. Revised 508 and EN 301 549 standards.  Any changes should be closely scrutinized.
5. This is a general comment regarding the incorporation of the WCAG and WCAG2ICT language into this WCAG2Mobile document. I do recall that Alastair liked this incorporation - even when they have no changes/additions in WCAG2Mobile. However, the WCAG2ICT content also includes interpretation for non-web documents so it seems a bit odd to include those parts verbatim. I'm ambivalent, but do recognize that incorporation of the WCAG2ICT text makes the MATF document brittle and tied to a specific version of the WCAG2ICT Group Note - especially since we are on different release cycles. **Recommendation: Perhaps open an issue with a general comment that says:**>Incorporation of the WCAG2ICT language is unhelpful if the WCAG2Mobile interpretation differs in its scope or interpretation the criteria specifically for non-web software. This is particularly difficult in light of the fact that existing standards that are applying WCAG to all non-web software will be unable to easily resolve differences in interpretation any time in the near future because WCAG itself has also not been applying the criteria to "screens" or "views" - at least not until WCAG 3 becomes a Recommendation.
5. It seems that the suggested note places a requirement on software platforms.  While I agree there should be such a requirement placed on platform software, it can't be done in a note and would have to be done outside of the WCAG2ICT document itself. IMO, if we suggest adding a note (or adding to the other note) perhaps it should be worded in a way that doesn't sound like it's placing a requirement on platform software: 
>NOTE: Non-web documents and software can only support programmatic labels for non-web content to the extent that the platform software provides that capability and exposes it to assistive technology.
